Author: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Date: Wed, 3 Feb 2016 00:22:52 +0000
Cc: 669053@bugs.debian.org
Bug-Debian: http://bugs.debian.org/669053
Description: Various manpage improvements.

Enable and fix warnings from 'man' and 'groff'.

Change `` to ".

Split lines longer than about 80 characters into two or more
lines.  References:

  1) man-pages(7) from package \"man-pages\" or
  \"www.kernel.org/doc/man-pages\" section 7 or
  \"man7.org/linux/man-pages/man7/man-pages.7.html\":

  New sentences should be started on new lines.
  This makes it easier to see the effect of patches,
  which often operate at the level of individual sentences.

Remove space at end of lines.

The space between sentences in "roff" is two spaces.

Better is to begin each sentence on a new line to avoid different
writers' conventions.

Change \' (acute) to \(aq, if used as a quote.

Change \' (acute) to ', if used as an apostrophe.

Change - to \(en (en-dash) for a numeric range.

Protect a full stop (.) with \&, if

  a) it can be transported to the first column, when the line is
     reformatted (split).
  b) it is the last part of an abbreviation that does not end a
     sentence.

Add a (no-break, "\ ") space between a number and an unit as these
are not one entity

Change a hyphen-minus to a minus (\-), if in front of a name for an option

Remove space in the first column, if not intended.

diff --git a/smartd.conf.5.in b/smartd.conf.5.in
index 21b031f..df2b6c4 100644
--- a/smartd.conf.5.in
+++ b/smartd.conf.5.in
@@ -15,7 +15,7 @@ You should have received a copy of the GNU General Public License
 This code was originally developed as a Senior Thesis by Michael Cornwell
 at the Concurrent Systems Laboratory (now part of the Storage Systems
 Research Center), Jack Baskin School of Engineering, University of
-California, Santa Cruz. http://ssrc.soe.ucsc.edu/
+California, Santa Cruz.  http://ssrc.soe.ucsc.edu/
 
 ..
 .TH SMARTD.CONF 5 "CURRENT_SVN_DATE" "CURRENT_SVN_VERSION" "SMART Monitoring Tools"
@@ -33,23 +33,23 @@ daemon.
 
 If the configuration file \fB/usr/local/etc/smartd.conf\fP is present,
 \fBsmartd\fP reads it at startup, before \fBfork\fP(2)ing into the
-background. If \fBsmartd\fP subsequently receives a \fBHUP\fP signal,
+background.  If \fBsmartd\fP subsequently receives a \fBHUP\fP signal,
 it will then re-read the configuration file.  If \fBsmartd\fP is
 running in debug mode, then an \fBINT\fP signal will also make it
-re-read the configuration file. This signal can be generated by typing
-\fB\<CONTROL-C\>\fP in the terminal window where \fBsmartd\fP is
+re-read the configuration file.  This signal can be generated by typing
+\fB<CONTROL-C>\fP in the terminal window where \fBsmartd\fP is
 running.
 
 In the absence of a configuration file
 \fBsmartd\fP will try to open all available devices
 (see \fBsmartd\fP(8) man page).
-A configuration file with a single line \fB\'DEVICESCAN \-a'\fP
+A configuration file with a single line \fB\(aqDEVICESCAN \-a\(aq\fP
 would have the same effect.
 
 This can be annoying if you have an ATA or SCSI device that hangs or
 misbehaves when receiving SMART commands.  Even if this causes no
 problems, you may be annoyed by the string of error log messages about devices
-that can\'t be opened.
+that can't be opened.
 
 One can avoid this problem, and gain more control over the types of
 events monitored by
@@ -59,27 +59,27 @@ by using the configuration file
 This file contains a list of devices to monitor, with one device per
 line.  An example file is included with the
 .B smartmontools
-distribution. You will find this sample configuration file in
-\fB/usr/local/share/doc/smartmontools/\fP. For security, the configuration file
-should not be writable by anyone but root. The syntax of the file is as
+distribution.  You will find this sample configuration file in
+\fB/usr/local/share/doc/smartmontools/\fP.  For security, the configuration file
+should not be writable by anyone but root.  The syntax of the file is as
 follows:
 .IP \(bu 4
 There should be one device listed per line, although you may have
 lines that are entirely comments or white space.
 .IP \(bu 4
-Any text following a hash sign \'#\' and up to the end of the line is
+Any text following a hash sign \(aq#\(aq and up to the end of the line is
 taken to be a comment, and ignored.
 .IP \(bu 4
-Lines may be continued by using a backslash \'\e\' as the last
+Lines may be continued by using a backslash \(aq\e\(aq as the last
 non-whitespace or non-comment item on a line.
 .IP \(bu 4
-Note: a line whose first character is a hash sign \'#\' is treated as
+Note: a line whose first character is a hash sign \(aq#\(aq is treated as
 a white-space blank line, \fBnot\fP as a non-existent line, and will
 \fBend\fP a continuation line.
 .PP
 
-Here is an example configuration file.  It\'s for illustrative purposes
-only; please don\'t copy it onto your system without reading to the end
+Here is an example configuration file.  It's for illustrative purposes
+only; please don't copy it onto your system without reading to the end
 of the
 .B DIRECTIVES
 Section below!
@@ -92,127 +92,127 @@ Section below!
 .B # On the second disk, start a long self-test every
 .B # Sunday between 3 and 4 am.
 .B #
-.B \ \ /dev/sda -a -m admin@example.com,root@localhost
-.B \ \ /dev/sdb -a -I 194 -I 5 -i 12 -s L/../../7/03
+.B \ \ /dev/sda \-a \-m admin@example.com,root@localhost
+.B \ \ /dev/sdb \-a \-I 194 \-I 5 \-i 12 \-s L/../../7/03
 .B #
 .B # Send a TEST warning email to admin on startup.
 .B #
-.B \ \ /dev/sdc -m admin@example.com -M test
+.B \ \ /dev/sdc \-m admin@example.com \-M test
 .B #
-.B # Strange device.  It\'s SCSI. Start a scheduled
+.B # Strange device.  It's SCSI.  Start a scheduled
 .B # long self test between 5 and 6 am Monday/Thursday
-.B \ \ /dev/weird -d scsi -s L/../../(1|4)/05
+.B \ \ /dev/weird \-d scsi \-s L/../../(1|4)/05
 .B #
 .B # An ATA disk may appear as a SCSI device to the
-.B # OS. If a SCSI to ATA Translation (SAT) layer
+.B # OS.  If a SCSI to ATA Translation (SAT) layer
 .B # is between the OS and the device then this can be
-.B # flagged with the '-d sat' option. This situation
+.B # flagged with the '-d sat' option.  This situation
 .B # may become common with SATA disks in SAS and FC
 .B # environments.
-.B \ \ /dev/sda -a -d sat
+.B \ \ /dev/sda \-a \-d sat
 .B #
 .\" %IF OS Linux
 .B # Three disks connected to a MegaRAID controller
-.B # Start short self-tests daily between 1-2, 2-3, and
-.B # 3-4 am.
-.B \ \ /dev/sda -d megaraid,0 -a -s S/../.././01
-.B \ \ /dev/sda -d megaraid,1 -a -s S/../.././02
-.B \ \ /dev/sda -d megaraid,2 -a -s S/../.././03
-.B \ \ /dev/bus/0 -d megaraid,2 -a -s S/../.././03
+.B # Start short self-tests daily between 1\(en2, 2\(en3, and
+.B # 3\(en4 am.
+.B \ \ /dev/sda \-d megaraid,0 \-a \-s S/../.././01
+.B \ \ /dev/sda \-d megaraid,1 \-a \-s S/../.././02
+.B \ \ /dev/sda \-d megaraid,2 \-a \-s S/../.././03
+.B \ \ /dev/bus/0 \-d megaraid,2 \-a \-s S/../.././03
 .B #
 .B # Three disks connected to an AacRaid controller
-.B # Start short self-tests daily between 1-2, 2-3, and
-.B # 3-4 am.
-.B \ \ /dev/sda -d aacraid,0,0,66 -a -s S/../.././01
-.B \ \ /dev/sda -d aacraid,0,0,67 -a -s S/../.././02
-.B \ \ /dev/sda -d aacraid,0,0,68 -a -s S/../.././03
+.B # Start short self-tests daily between 1\(en2, 2\(en3, and
+.B # 3\(en4 am.
+.B \ \ /dev/sda \-d aacraid,0,0,66 \-a \-s S/../.././01
+.B \ \ /dev/sda \-d aacraid,0,0,67 \-a \-s S/../.././02
+.B \ \ /dev/sda \-d aacraid,0,0,68 \-a \-s S/../.././03
 .B #
 .\" %ENDIF OS Linux
 .B # Four ATA disks on a 3ware 6/7/8000 controller.
-.B # Start short self-tests daily between midnight and 1am,
-.B # 1-2, 2-3, and 3-4 am.  Starting with the Linux 2.6
+.B # Start short self-tests daily between midnight and 1 am,
+.B # 1\(en2, 2\(en3, and 3\(en4 am.  Starting with the Linux 2.6
 .B # kernel series, /dev/sdX is deprecated in favor of
 .B # /dev/tweN.  For example replace /dev/sdc by /dev/twe0
 .B # and /dev/sdd by /dev/twe1.
-.B \ \ /dev/sdc -d 3ware,0 -a -s S/../.././00
-.B \ \ /dev/sdc -d 3ware,1 -a -s S/../.././01
-.B \ \ /dev/sdd -d 3ware,2 -a -s S/../.././02
-.B \ \ /dev/sdd -d 3ware,3 -a -s S/../.././03
+.B \ \ /dev/sdc \-d 3ware,0 \-a \-s S/../.././00
+.B \ \ /dev/sdc \-d 3ware,1 \-a \-s S/../.././01
+.B \ \ /dev/sdd \-d 3ware,2 \-a \-s S/../.././02
+.B \ \ /dev/sdd \-d 3ware,3 \-a \-s S/../.././03
 .B #
 .B # Two ATA disks on a 3ware 9000 controller.
 .B # Start long self-tests Sundays between midnight and
-.B # 1am and 2-3 am
-.B \ \ /dev/twa0 -d 3ware,0 -a -s L/../../7/00
-.B \ \ /dev/twa0 -d 3ware,1 -a -s L/../../7/02
+.B # 1 am and 2\(en3 am
+.B \ \ /dev/twa0 \-d 3ware,0 \-a \-s L/../../7/00
+.B \ \ /dev/twa0 \-d 3ware,1 \-a \-s L/../../7/02
 .B #
 .B # Two SATA (not SAS) disks on a 3ware 9750 controller.
 .B # Start long self-tests Sundays between midnight and
-.B # 1am and 2-3 am
+.B # 1 am and 2\(en3 am
 .\" %IF OS Linux
-.B \ \ /dev/twl0 -d 3ware,0 -a -s L/../../7/00
-.B \ \ /dev/twl0 -d 3ware,1 -a -s L/../../7/02
+.B \ \ /dev/twl0 \-d 3ware,0 \-a \-s L/../../7/00
+.B \ \ /dev/twl0 \-d 3ware,1 \-a \-s L/../../7/02
 .\" %ENDIF OS Linux
 .\" %IF OS FreeBSD
-.B \ \ /dev/tws0 -d 3ware,0 -a -s L/../../7/00
-.B \ \ /dev/tws0 -d 3ware,1 -a -s L/../../7/02
+.B \ \ /dev/tws0 \-d 3ware,0 \-a \-s L/../../7/00
+.B \ \ /dev/tws0 \-d 3ware,1 \-a \-s L/../../7/02
 .\" %ENDIF OS FreeBSD
 .B #
 .B # Three SATA disks on a HighPoint RocketRAID controller.
-.B # Start short self-tests daily between 1-2, 2-3, and
-.B # 3-4 am.
+.B # Start short self-tests daily between 1\(en2, 2\(en3, and
+.B # 3\(en4 am.
 .\" %IF OS Linux
 .B # under Linux
-.B \ \ /dev/sde -d hpt,1/1 -a -s S/../.././01
-.B \ \ /dev/sde -d hpt,1/2 -a -s S/../.././02
-.B \ \ /dev/sde -d hpt,1/3 -a -s S/../.././03
+.B \ \ /dev/sde \-d hpt,1/1 \-a \-s S/../.././01
+.B \ \ /dev/sde \-d hpt,1/2 \-a \-s S/../.././02
+.B \ \ /dev/sde \-d hpt,1/3 \-a \-s S/../.././03
 .\" %ENDIF OS Linux
 .\" %IF OS FreeBSD
 .B # under FreeBSD
-.B  /dev/hptrr -d hpt,1/1 -a -s S/../.././01
-.B  /dev/hptrr -d hpt,1/2 -a -s S/../.././02
-.B  /dev/hptrr -d hpt,1/3 -a -s S/../.././03
+.B  /dev/hptrr \-d hpt,1/1 \-a \-s S/../.././01
+.B  /dev/hptrr \-d hpt,1/2 \-a \-s S/../.././02
+.B  /dev/hptrr \-d hpt,1/3 \-a \-s S/../.././03
 .\" %ENDIF OS FreeBSD
 .B #
-.B # Two SATA disks connected to a HighPoint RocketRAID 
+.B # Two SATA disks connected to a HighPoint RocketRAID
 .B # via a pmport device.  Start long self-tests Sundays
-.B # between midnight and 1am and 2-3 am.
+.B # between midnight and 1 am and 2\(en3 am.
 .\" %IF OS Linux
 .B # under Linux
-.B \ \ /dev/sde -d hpt,1/4/1 -a -s L/../../7/00
-.B \ \ /dev/sde -d hpt,1/4/2 -a -s L/../../7/02
+.B \ \ /dev/sde \-d hpt,1/4/1 \-a \-s L/../../7/00
+.B \ \ /dev/sde \-d hpt,1/4/2 \-a \-s L/../../7/02
 .\" %ENDIF OS Linux
 .\" %IF OS FreeBSD
 .B # under FreeBSD
-.B \ \ /dev/hptrr -d hpt,1/4/1 -a -s L/../../7/00
-.B \ \ /dev/hptrr -d hpt,1/4/2 -a -s L/../../7/02
+.B \ \ /dev/hptrr \-d hpt,1/4/1 \-a \-s L/../../7/00
+.B \ \ /dev/hptrr \-d hpt,1/4/2 \-a \-s L/../../7/02
 .B #
 .\" %ENDIF OS FreeBSD
 .B # Three SATA disks connected to an Areca
-.B # RAID controller. Start long self-tests Sundays
+.B # RAID controller.  o51Start long self-tests Sundays
 .B # between midnight and 3 am.
 .\" %IF OS Linux
-.B \ \ /dev/sg2 -d areca,1 -a -s L/../../7/00
-.B \ \ /dev/sg2 -d areca,2 -a -s L/../../7/01
-.B \ \ /dev/sg2 -d areca,3 -a -s L/../../7/02
+.B \ \ /dev/sg2 \-d areca,1 \-a \-s L/../../7/00
+.B \ \ /dev/sg2 \-d areca,2 \-a \-s L/../../7/01
+.B \ \ /dev/sg2 \-d areca,3 \-a \-s L/../../7/02
 .\" %ENDIF OS Linux
 .\" %IF OS FreeBSD
-.B \ \ /dev/arcmsr0 -d areca,1 -a -s L/../../7/00
-.B \ \ /dev/arcmsr0 -d areca,2 -a -s L/../../7/01
-.B \ \ /dev/arcmsr0 -d areca,3 -a -s L/../../7/02
+.B \ \ /dev/arcmsr0 \-d areca,1 \-a \-s L/../../7/00
+.B \ \ /dev/arcmsr0 \-d areca,2 \-a \-s L/../../7/01
+.B \ \ /dev/arcmsr0 \-d areca,3 \-a \-s L/../../7/02
 .\" %ENDIF OS FreeBSD
 .B #
-.B # The following line enables monitoring of the 
-.B # ATA Error Log and the Self-Test Error Log.  
+.B # The following line enables monitoring of the
+.B # ATA Error Log and the Self-Test Error Log.
 .B # It also tracks changes in both Prefailure
 .B # and Usage Attributes, apart from Attributes
 .B # 9, 194, and 231, and shows  continued lines:
 .B #
-.B \ \ /dev/sdd\ -l\ error\ \e
-.B \ \ \ \ \ \ \ \ \ \ \ -l\ selftest\ \e
-.B \ \ \ \ \ \ \ \ \ \ \ -t\ \e\ \ \ \ \ \ # Attributes not tracked:
-.B \ \ \ \ \ \ \ \ \ \ \ -I\ 194\ \e\ \ # temperature
-.B \ \ \ \ \ \ \ \ \ \ \ -I\ 231\ \e\ \ # also temperature
-.B \ \ \ \ \ \ \ \ \ \ \ -I 9\ \ \ \ \ \ # power-on hours
+.B \ \ /dev/sdd\ \-l\ error\ \e
+.B \ \ \ \ \ \ \ \ \ \ \ \-l\ selftest\ \e
+.B \ \ \ \ \ \ \ \ \ \ \ \-t\ \e\ \ \ \ \ \ # Attributes not tracked:
+.B \ \ \ \ \ \ \ \ \ \ \ \-I\ 194\ \e\ \ # temperature
+.B \ \ \ \ \ \ \ \ \ \ \ \-I\ 231\ \e\ \ # also temperature
+.B \ \ \ \ \ \ \ \ \ \ \ \-I 9\ \ \ \ \ \ # power-on hours
 .B #
 .B ################################################
 .fi
@@ -237,23 +237,23 @@ as defaults for the next device entries.
 This configuration:
 .PP
 .nf
-\ \ DEFAULT -a -R5! -W 2,40,45 -I 194 -s L/../../7/00 -m admin@example.com
+\ \ DEFAULT \-a \-R5! \-W 2,40,45 \-I 194 \-s L/../../7/00 \-m admin@example.com
 \ \ /dev/sda
 \ \ /dev/sdb
 \ \ /dev/sdc
-\ \ DEFAULT -H -m admin@example.com
+\ \ DEFAULT \-H \-m admin@example.com
 \ \ /dev/sdd
-\ \ /dev/sde -d removable
+\ \ /dev/sde \-d removable
 .fi
 .PP
 has the same effect as:
 .PP
 .nf
-\ \ /dev/sda -a -R5! -W 2,40,45 -I 194 -s L/../../7/00 -m admin@example.com
-\ \ /dev/sdb -a -R5! -W 2,40,45 -I 194 -s L/../../7/00 -m admin@example.com
-\ \ /dev/sdc -a -R5! -W 2,40,45 -I 194 -s L/../../7/00 -m admin@example.com
-\ \ /dev/sdd -H -m admin@example.com
-\ \ /dev/sde -d removable -H -m admin@example.com
+\ \ /dev/sda \-a \-R5! \-W 2,40,45 \-I 194 \-s L/../../7/00 \-m admin@example.com
+\ \ /dev/sdb \-a \-R5! \-W 2,40,45 \-I 194 \-s L/../../7/00 \-m admin@example.com
+\ \ /dev/sdc \-a \-R5! \-W 2,40,45 \-I 194 \-s L/../../7/00 \-m admin@example.com
+\ \ /dev/sdd \-H \-m admin@example.com
+\ \ /dev/sde \-d removable \-H \-m admin@example.com
 .fi
 
 
@@ -264,36 +264,36 @@ or
 .B DEFAULT
 on any line of the
 .B /usr/local/etc/smartd.conf
-configuration file. Note that
-.B these are NOT command-line options for 
+configuration file.  Note that
+.B these are NOT command-line options for
 \fBsmartd\fP.
 The Directives below may appear in any order, following the device
-name. 
+name.
 
 .B For an ATA device,
 if no Directives appear, then the device will be monitored
-as if the \'\-a\' Directive (monitor all SMART properties) had been given.
+as if the \(aq\-a\(aq Directive (monitor all SMART properties) had been given.
 
 .B If a SCSI disk is listed,
 it will be monitored at the maximum implemented level: roughly
-equivalent to using the \'\-H \-l selftest\' options for an ATA disk.
-So with the exception of \'\-d\', \'\-m\', \'\-l selftest\', \'\-s\', and
-\'\-M\', the Directives below are ignored for SCSI disks.  For SCSI
-disks, the \'\-m\' Directive sends a warning email if the SMART status
+equivalent to using the \(aq\-H \-l selftest\(aq options for an ATA disk.
+So with the exception of \(aq\-d\(aq, \(aq\-m\(aq, \(aq\-l selftest\(aq, \(aq\-s\(aq, and
+\(aq\-M\(aq, the Directives below are ignored for SCSI disks.  For SCSI
+disks, the \(aq\-m\(aq Directive sends a warning email if the SMART status
 indicates a disk failure or problem, if the SCSI inquiry about disk
 status fails, or if new errors appear in the self-test log.
 
 .B If a 3ware controller is used
 then the corresponding SCSI (/dev/sd?) or character device (/dev/twe?,
-/dev/twa?, /dev/twl? or /dev/tws?) must be listed, along with the \'\-d 3ware,N\'
+/dev/twa?, /dev/twl? or /dev/tws?) must be listed, along with the \(aq\-d 3ware,N\(aq
 Directive (see below).  The individual ATA disks hosted by the 3ware
 controller appear to \fBsmartd\fP as normal ATA devices.  Hence all
 the ATA directives can be used for these disks (but see note below).
 
 .\" %IF OS Linux FreeBSD
 .B If an Areca controller is used
-then the corresponding device (SCSI /dev/sg? on Linux or /dev/arcmsr0 on 
-FreeBSD) must be listed, along with the \'\-d areca,N\' Directive (see below).
+then the corresponding device (SCSI /dev/sg? on Linux or /dev/arcmsr0 on
+FreeBSD) must be listed, along with the \(aq\-d areca,N\(aq Directive (see below).
 The individual SATA disks hosted by the Areca controller appear to \fBsmartd\fP
 as normal ATA devices.  Hence all the ATA directives can be used for
 these disks.  Areca firmware version 1.46 or later which supports
@@ -328,9 +328,9 @@ This is for ATA disks that have a SCSI to ATA Translation (SAT) Layer
 (SATL) between the disk and the operating system.
 SAT defines two ATA PASS THROUGH SCSI commands, one 12 bytes long and
 the other 16 bytes long.  The default is the 16 byte variant which can be
-overridden with either \'\-d sat,12\' or \'\-d sat,16\'.
+overridden with either \(aq\-d sat,12\(aq or \(aq\-d sat,16\(aq.
 
-If \'\-d sat,auto\' is specified, device type SAT (for ATA/SATA disks) is
+If \(aq\-d sat,auto\(aq is specified, device type SAT (for ATA/SATA disks) is
 only used if the SCSI INQUIRY data reports a SATL (VENDOR: "ATA     ").
 Otherwise device type SCSI (for SCSI/SAS disks) is used.
 
@@ -338,28 +338,28 @@ Otherwise device type SCSI (for SCSI/SAS disks) is used.
 \- this device type is for ATA disks that are behind a Cypress USB to PATA
 bridge.  This will use the ATACB proprietary scsi pass through command.
 The default SCSI operation code is 0x24, but although it can be overridden
-with \'\-d usbcypress,0xN\', where N is the scsi operation code,
+with \(aq\-d usbcypress,0xN\(aq, where N is the scsi operation code,
 you're running the risk of damage to the device or filesystems on it.
 
 .I usbjmicron[,p][,x][,PORT]
 \- this device type is for SATA disks that are behind a JMicron USB to
-PATA/SATA bridge.  The 48-bit ATA commands (required e.g. for \'\-l xerror\',
+PATA/SATA bridge.  The 48-bit ATA commands (required e.g.\& for \(aq\-l xerror\(aq,
 see below) do not work with all of these bridges and are therefore disabled by
-default.  These commands can be enabled by \'\-d usbjmicron,x\'.
+default.  These commands can be enabled by \(aq\-d usbjmicron,x\(aq.
 If two disks are connected to a bridge with two ports, an error message is printed
 if no PORT is specified.
-The port can be specified by \'\-d usbjmicron[,x],PORT\' where PORT is 0
+The port can be specified by \(aq\-d usbjmicron[,x],PORT\(aq where PORT is 0
 (master) or 1 (slave).  This is not necessary if the device uses a port
 multiplier to connect multiple disks to one port.  The disks appear under
 separate /dev/ice names then.
-CAUTION: Specifying \',x\' for a device which does not support it results
+CAUTION: Specifying \(aq,x\(aq for a device which does not support it results
 in I/O errors and may disconnect the drive.  The same applies if the specified
 PORT does not exist or is not connected to a disk.
 
 The Prolific PL2507/3507 USB bridges with older firmware support a pass-through
-command similar to JMicron and work with \'\-d usbjmicron,0\'.
+command similar to JMicron and work with \(aq\-d usbjmicron,0\(aq.
 Newer Prolific firmware requires a modified command which can be selected by
-\'\-d usbjmicron,p\'.
+\(aq\-d usbjmicron,p\(aq.
 Note that this does not yet support the SMART status command.
 
 .I usbprolific
@@ -384,7 +384,7 @@ to a MegaRAID controller.  The non-negative integer N (in the range of 0 to
 This interface will also work for Dell PERC controllers.
 In log files and email messages this disk will be identified as
 megaraid_disk_XXX with XXX in the range from 000 to 127 inclusive.
-It is possible to set RAID device name as /dev/bus/N, where N is a SCSI bus 
+It is possible to set RAID device name as /dev/bus/N, where N is a SCSI bus
 number.
 Please see the \fBsmartctl\fP(8) man page for further details.
 
@@ -420,7 +420,7 @@ Please see the \fBsmartctl\fP(8) man page for further details.
 \- [FreeBSD, Linux, Windows and Cygwin only] the device consists of one or more SATA disks
 connected to an Areca SATA RAID controller.  The positive integer N (in the range
 from 1 to 24 inclusive) denotes which disk on the controller is monitored.
-In log files and email messages this disk will be identifed as
+In log files and email messages this disk will be identified as
 areca_disk_XX with XX in the range from 01 to 24 inclusive.
 Please see the \fBsmartctl\fP(8) man page for further details.
 
@@ -460,7 +460,7 @@ Please see the \fBsmartctl\fP(8) man page for further details.
 This allows to ignore specific devices which are detected by a following
 DEVICESCAN configuration line.
 It may also be used to temporary disable longer multi-line configuration entries.
-This Directive may be used in conjunction with the other \'\-d\' Directives.
+This Directive may be used in conjunction with the other \(aq\-d\(aq Directives.
 
 .I removable
 \- the device or its media is removable.  This indicates to
@@ -468,16 +468,16 @@ This Directive may be used in conjunction with the other \'\-d\' Directives.
 that it should continue (instead of exiting, which is the default
 behavior) if the device does not appear to be present when
 \fBsmartd\fP is started.  This Directive may be used in conjunction
-with the other \'\-d\' Directives.
+with the other \(aq\-d\(aq Directives.
 .TP
 .B \-n POWERMODE[,N][,q]
-[ATA only] This \'nocheck\' Directive is used to prevent a disk from
+[ATA only] This \(aqnocheck\(aq Directive is used to prevent a disk from
 being spun-up when it is periodically polled by \fBsmartd\fP.
 
-ATA disks have five different power states. In order of increasing
-power consumption they are: \'OFF\', \'SLEEP\', \'STANDBY\', \'IDLE\',
-and \'ACTIVE\'.  Typically in the OFF, SLEEP, and STANDBY modes the
-disk\'s platters are not spinning. But usually, in response to SMART
+ATA disks have five different power states.  In order of increasing
+power consumption they are: \(aqOFF\(aq, \(aqSLEEP\(aq, \(aqSTANDBY\(aq, \(aqIDLE\(aq,
+and \(aqACTIVE\(aq.  Typically in the OFF, SLEEP, and STANDBY modes the
+disk's platters are not spinning.  But usually, in response to SMART
 commands issued by \fBsmartd\fP, the disk platters are spun up.  So if
 this option is not used, then a disk which is in a low-power mode may
 be spun up and put into a higher-power mode when it is periodically
@@ -485,11 +485,11 @@ polled by \fBsmartd\fP.
 
 Note that if the disk is in SLEEP mode when \fBsmartd\fP is started,
 then it won't respond to \fBsmartd\fP commands, and so the disk won't
-be registered as a device for \fBsmartd\fP to monitor. If a disk is in
+be registered as a device for \fBsmartd\fP to monitor.  If a disk is in
 any other low-power mode, then the commands issued by \fBsmartd\fP to
 register the disk will probably cause it to spin-up.
 
-The \'\fB\-n\fP\' (nocheck) Directive specifies if \fBsmartd\fP\'s
+The \(aq\fB\-n\fP\(aq (nocheck) Directive specifies if \fBsmartd\fP's
 periodic checks should still be carried out when the device is in a
 low-power mode.  It may be used to prevent a disk from being spun-up
 by periodic \fBsmartd\fP polling.  The allowed values of POWERMODE
@@ -497,7 +497,7 @@ are:
 
 .I never
 \- \fBsmartd\fP will poll (check) the device regardless of its power
-mode. This may cause a disk which is spun-down to be spun-up when
+mode.  This may cause a disk which is spun-down to be spun-up when
 \fBsmartd\fP checks it.  This is the default behavior if the '\-n'
 Directive is not given.
 
@@ -516,16 +516,16 @@ In the IDLE state, most disks are still spinning, so this is probably
 not what you want.
 
 Maximum number of skipped checks (in a row) can be specified by
-appending positive number \',N\' to POWERMODE (like \'\-n standby,15\').
+appending positive number \(aq,N\(aq to POWERMODE (like \(aq\-n standby,15\(aq).
 After N checks are skipped in a row, powermode is ignored and the
 check is performed anyway.
 
 When a periodic test is skipped, \fBsmartd\fP normally writes an
-informal log message. The message can be suppressed by appending
-the option \',q\' to POWERMODE (like \'\-n standby,q\').
+informal log message.  The message can be suppressed by appending
+the option \(aq,q\(aq to POWERMODE (like \(aq\-n standby,q\(aq).
 This prevents a laptop disk from spinning up due to this message.
 
-Both \',N\' and \',q\' can be specified together.
+Both \(aq,N\(aq and \(aq,q\(aq can be specified together.
 .TP
 .B \-T TYPE
 Specifies how tolerant
@@ -570,7 +570,7 @@ Directive are \fIon\fP and \fIoff\fP.  Also affects SCSI devices.
 STATUS command.
 If this command reports a failing health status, then disk
 failure is predicted in less than 24 hours, and a message at loglevel
-.B \'LOG_CRIT\'
+.B \(aqLOG_CRIT\(aq
 will be logged to syslog.  [Please see the
 .B smartctl \-H
 command-line option.]
@@ -587,7 +587,7 @@ error log has increased since the last check.
 \- [ATA only] report if the number of ATA errors reported in the Extended
 Comprehensive SMART error log has increased since the last check.
 
-If both \'\-l error\' and \'\-l xerror\' are specified, smartd checks
+If both \(aq\-l error\(aq and \(aq\-l xerror\(aq are specified, smartd checks
 the maximum of both values.
 
 [Please see the \fBsmartctl \-l xerror\fP command-line option.]
@@ -598,10 +598,10 @@ Self-Test Log has increased since the last check, or if the timestamp
 associated with the most recent failed test has increased.  Note that
 such errors will \fBonly\fP be logged if you run self-tests on the
 disk (and it fails a test!).  Self-Tests can be run automatically by
-\fBsmartd\fP: please see the \fB\'\-s\'\fP Directive below.
-Self-Tests can also be run manually by using the \fB\'\-t\ short\'\fP
-and \fB\'\-t\ long\'\fP options of \fBsmartctl\fP and the results of
-the testing can be observed using the \fBsmartctl \'\-l\ selftest\'\fP
+\fBsmartd\fP: please see the \fB\(aq\-s\(aq\fP Directive below.
+Self-Tests can also be run manually by using the \fB\(aq\-t\ short\(aq\fP
+and \fB\(aq\-t\ long\(aq\fP options of \fBsmartctl\fP and the results of
+the testing can be observed using the \fBsmartctl \(aq\-l\ selftest\(aq\fP
 command-line option.
 [Please see the \fBsmartctl \-l\fP and \fB\-t\fP command-line
 options.]
@@ -615,16 +615,16 @@ an extended self-test is run after all bad sectors have been reallocated.
 \- [ATA only] report if the Offline Data Collection status has changed
 since the last check.  The report will be logged as LOG_CRIT if the new
 status indicates an error.  With some drives the status often changes,
-therefore \'\-l offlinests\' is not enabled by '\-a\' Directive.
+therefore \(aq\-l offlinests\(aq is not enabled by \(aq\-a\(aq Directive.
 .\" %IF NOT OS Cygwin Windows
-.\"! Appending \',ns\' (no standby) to this directive is not implemented
+.\"! Appending ',ns' (no standby) to this directive is not implemented
 .\"! on OS_MAN_FILTER.
 .\" %ENDIF NOT OS Cygwin Windows
 .\" %IF OS Cygwin Windows
 
-[Windows and Cygwin only] If \',ns\' (no standby) is appended to this
+[Windows and Cygwin only] If \(aq,ns\(aq (no standby) is appended to this
 directive, smartd disables system auto standby as long as an Offline
-Data Collection is in progress. See \'\-l selfteststs,ns\' below.
+Data Collection is in progress.  See \(aq\-l selfteststs,ns\(aq below.
 .\" %ENDIF OS Cygwin Windows
 
 .I selfteststs[,ns]
@@ -632,16 +632,16 @@ Data Collection is in progress. See \'\-l selfteststs,ns\' below.
 since the last check.  The report will be logged as LOG_CRIT if the new
 status indicates an error.
 .\" %IF NOT OS Cygwin Windows
-.\"! Appending \',ns\' (no standby) to this directive is not implemented
+.\"! Appending ',ns' (no standby) to this directive is not implemented
 .\"! on OS_MAN_FILTER.
 .\" %ENDIF NOT OS Cygwin Windows
 .\" %IF OS Cygwin Windows
 
-[Windows and Cygwin only] If \',ns\' (no standby) is appended to this
+[Windows and Cygwin only] If \(aq,ns\(aq (no standby) is appended to this
 directive, smartd disables system auto standby as long as a Self-Test
 is in progress.  This prevents that a Self-Test is aborted because the
 OS sets the system to a standby/sleep mode when idle.  Smartd check
-interval (\'\-i\' option) should be shorter than the configured idle
+interval (\(aq\-i\(aq option) should be shorter than the configured idle
 timeout.  Auto standby is not disabled if the system is running on
 battery.
 .\" %ENDIF OS Cygwin Windows
@@ -683,25 +683,25 @@ IDLE mode.
 Run Self-Tests or Offline Immediate Tests, at scheduled times.  A
 Self- or Offline Immediate Test will be run at the end of periodic
 device polling, if all 12 characters of the string \fBT/MM/DD/d/HH\fP
-match the extended regular expression \fBREGEXP\fP. Here:
+match the extended regular expression \fBREGEXP\fP.  Here:
 .RS 7
 .IP \fBT\fP 4
 is the type of the test.  The values that \fBsmartd\fP will try to
-match (in turn) are: \'L\' for a \fBL\fPong Self-Test, \'S\' for a
-\fBS\fPhort Self-Test, \'C\' for a \fBC\fPonveyance Self-Test (ATA
-only), and \'O\' for an \fBO\fPffline Immediate Test (ATA only).  As
+match (in turn) are: \(aqL\(aq for a \fBL\fPong Self-Test, \(aqS\(aq for a
+\fBS\fPhort Self-Test, \(aqC\(aq for a \fBC\fPonveyance Self-Test (ATA
+only), and \(aqO\(aq for an \fBO\fPffline Immediate Test (ATA only).  As
 soon as a match is found, the test will be started and no additional
 matches will be sought for that device and that polling cycle.
 
-To run scheduled Selective Self-Tests, use \'n\' for \fBn\fPext span,
-\'r\' to \fBr\fPedo last span, or \'c\' to \fBc\fPontinue with next span
+To run scheduled Selective Self-Tests, use \(aqn\(aq for \fBn\fPext span,
+\(aqr\(aq to \fBr\fPedo last span, or \(aqc\(aq to \fBc\fPontinue with next span
 or redo last span based on status of last test.
 The LBA range is based on the first span from the last test.
 See the \fBsmartctl \-t select,[next|redo|cont]\fP options for
 further info.
 
-Some disks (e.g. WD) do not preserve the selective self test log accross
-power cycles.  If state persistence (\'\-s\' option) is enabled, the last
+Some disks (e.g.\& WD) do not preserve the selective self test log across
+power cycles.  If state persistence (\(aq\-s\(aq option) is enabled, the last
 test span is preserved by smartd and used if (and only if) the selective
 self test log is empty.
 .IP \fBMM\fP 4
@@ -709,7 +709,7 @@ is the month of the year, expressed with two decimal digits.  The
 range is from 01 (January) to 12 (December) inclusive.  Do \fBnot\fP
 use a single decimal digit or the match will always fail!
 .IP \fBDD\fP 4
-is the day of the month, expressed with two decimal digits. The
+is the day of the month, expressed with two decimal digits.  The
 range is from 01 to 31 inclusive.  Do \fBnot\fP
 use a single decimal digit or the match will always fail!
 .IP \fBd\fP 4
@@ -717,7 +717,7 @@ is the day of the week, expressed with one decimal digit.  The
 range is from 1 (Monday) to 7 (Sunday) inclusive.
 .IP \fBHH\fP 4
 is the hour of the day, written with two decimal digits, and given in
-hours after midnight.  The range is 00 (midnight to just before 1am)
+hours after midnight.  The range is 00 (midnight to just before 1 am)
 to 23 (11pm to just before midnight) inclusive.  Do \fBnot\fP use a
 single decimal digit or the match will always fail!
 .RE
@@ -726,37 +726,37 @@ single decimal digit or the match will always fail!
 .TP
 .B \&
 Some examples follow.  In reading these, keep in mind that in extended
-regular expressions a dot \fB\'.\'\fP matches any single character, and
-a parenthetical expression such as \fB\'(A|B|C)\'\fP denotes any one of the three possibilities \fBA\fP,
-\fBB\fP, or \fBC\fP.
+regular expressions a dot \fB\(aq.\(aq\fP matches any single character, and
+a parenthetical expression such as \fB\(aq(A|B|C)\(aq\fP denotes any one
+of the three possibilities \fBA\fP, \fBB\fP, or \fBC\fP.
 
-To schedule a short Self-Test between 2-3am every morning, use:
+To schedule a short Self-Test between 2\(en3 am every morning, use:
 .nf
 \fB \-s S/../.././02\fP
 .fi
-To schedule a long Self-Test between 4-5am every Sunday morning, use:
+To schedule a long Self-Test between 4\(en5 am every Sunday morning, use:
 .nf
 \fB \-s L/../../7/04\fP
 .fi
-To schedule a long Self-Test between 10-11pm on the first and
+To schedule a long Self-Test between 10\(en11 pm on the first and
 fifteenth day of each month, use:
 .nf
 \fB \-s L/../(01|15)/./22\fP
 .fi
-To schedule an Offline Immediate test after every midnight, 6am,
-noon,and 6pm, plus a Short Self-Test daily at 1-2am and a Long
-Self-Test every Saturday at 3-4am, use:
+To schedule an Offline Immediate test after every midnight, 6 am,
+noon, and 6 pm, plus a Short Self-Test daily at 1\(en2 am and a Long
+Self-Test every Saturday at 3\(en4 am, use:
 .nf
 \fB \-s (O/../.././(00|06|12|18)|S/../.././01|L/../../6/03)\fP
 .fi
 If Long Self-Tests of a large disks take longer than the system uptime,
 a full disk test can be performed by several Selective Self-Tests.
-To setup a full test of a 1TB disk within 20 days (one 50GB span
+To setup a full test of a 1 TB disk within 20 days (one 50 GB span
 each day), run this command once:
 .nf
-  smartctl -t select,0-99999999 /dev/sda
+  smartctl \-t select,0-99999999 /dev/sda
 .fi
-To run the next test spans on Monday-Friday between 12-13am, run smartd
+To run the next test spans on Monday\(enFriday between 12\(en13 am, run smartd
 with this directive:
 .nf
 \fB \-s n/../../[1-5]/12\fP
@@ -767,7 +767,7 @@ Scheduled tests are run immediately following the regularly-scheduled
 device polling, if the current local date, time, and test type, match
 \fBREGEXP\fP.  By default the regularly-scheduled device polling
 occurs every thirty minutes after starting \fBsmartd\fP.  Take caution
-if you use the \'\-i\' option to make this polling interval more than
+if you use the \(aq\-i\(aq option to make this polling interval more than
 sixty minutes: the poll times may fail to coincide with any of the
 testing times that you have specified with \fBREGEXP\fP.  In this case
 the test will be run following the next device polling.
@@ -782,7 +782,7 @@ test was already started or run in the same hour.
 
 To avoid performance problems during system boot, \fBsmartd\fP will
 not attempt to run any scheduled tests following the very first
-device polling (unless \'\-q onecheck\' is specified).
+device polling (unless \(aq\-q onecheck\(aq is specified).
 
 Each time a test is run, \fBsmartd\fP will log an entry to SYSLOG.
 You can use these or the '-q showtests' command-line option to verify
@@ -792,12 +792,12 @@ if multiple test types are all scheduled for the same hour, the
 longer test type has precedence.  This is usually the desired behavior.
 
 If the scheduled tests are used in conjunction with state persistence
-(\'\-s\' option), smartd will also try to match the hours since last
-shutdown (or 90 days at most). If any test would have been started
+(\(aq\-s\(aq option), smartd will also try to match the hours since last
+shutdown (or 90 days at most).  If any test would have been started
 during downtime, the longest (see above) of these tests is run after
 second device polling.
 
-If the \'\-n\' directive is used and any test would have been started
+If the \(aq\-n\(aq directive is used and any test would have been started
 during disk standby time, the longest of these tests is run when the
 disk is active again.
 
@@ -809,24 +809,24 @@ in \fBREGEXP\fP that appear to indicate that you have made this
 mistake.
 .TP
 .B \-m ADD
-Send a warning email to the email address \fBADD\fP if the \'\-H\',
-\'\-l\', \'\-f\', \'\-C\', or \'\-O\' Directives detect a failure or a
-new error, or if a SMART command to the disk fails. This Directive
+Send a warning email to the email address \fBADD\fP if the \(aq\-H\(aq,
+\(aq\-l\(aq, \(aq\-f\(aq, \(aq\-C\(aq, or \(aq\-O\(aq Directives detect a failure or a
+new error, or if a SMART command to the disk fails.  This Directive
 only works in conjunction with these other Directives (or with the
-equivalent default \'\-a\' Directive).
+equivalent default \(aq\-a\(aq Directive).
 
 To prevent your email in-box from getting filled up with warning
 messages, by default only a single warning will be sent for each of
-the enabled alert types, \'\-H\', \'\-l\', \'\-f\', \'\-C\', or
-\'\-O\' even if more than one failure or error is detected or if the
+the enabled alert types, \(aq\-H\(aq, \(aq\-l\(aq, \(aq\-f\(aq, \(aq\-C\(aq, or
+\(aq\-O\(aq even if more than one failure or error is detected or if the
 failure or error persists.  [This behavior can be modified; see the
-\'\-M\' Directive below.]
+\(aq\-M\(aq Directive below.]
 
 To send email to more than one user, please use the following "comma
 separated" form for the address: \fBuser1@add1,user2@add2,...,userN@addN\fP
 (with no spaces).
 
-To test that email is being sent correctly, use the \'\-M test\'
+To test that email is being sent correctly, use the \(aq\-M test\(aq
 Directive described below to send one test email message on
 \fBsmartd\fP
 startup.
@@ -837,38 +837,38 @@ executable must be in the path of the shell or environment from which
 \fBsmartd\fP
 was started.  If you wish to specify an explicit path to the mail
 executable (for example /usr/local/bin/mail) or a custom script to
-run, please use the \'\-M exec\' Directive below.
+run, please use the \(aq\-M exec\(aq Directive below.
 
 .\" %IF OS Windows
-On Windows, the \'\fBBlat\fP\' mailer
+On Windows, the \(aq\fBBlat\fP\(aq mailer
 (\fBhttp://blat.sourceforge.net/\fP) is used by default.
 This mailer uses a different command line syntax, see
-\'\-M exec\' below.
+\(aq\-M exec\(aq below.
 
 .\" %ENDIF OS Windows
 Note also that there is a special argument
 .B <nomailer>
-which can be given to the \'\-m\' Directive in conjunction with the \'\-M
-exec\' Directive. Please see below for an explanation of its effect.
+which can be given to the \(aq\-m\(aq Directive in conjunction with the \(aq\-M
+exec\(aq Directive.  Please see below for an explanation of its effect.
 
 If the mailer or the shell running it produces any STDERR/STDOUT
 output, then a snippet of that output will be copied to SYSLOG.  The
-remainder of the output is discarded. If problems are encountered in
+remainder of the output is discarded.  If problems are encountered in
 sending mail, this should help you to understand and fix them.  If
 you have mail problems, we recommend running \fBsmartd\fP in debug
-mode with the \'-d\' flag, using the \'-M test\' Directive described
+mode with the \(aq\-d\(aq flag, using the \(aq\-M test\(aq Directive described
 below.
 .\" %IF ENABLE_SMARTDPLUGINDIR
 .\" %IF NOT OS Windows
 
-If a word of the comma separated list has the form \'@plugin\', a custom
+If a word of the comma separated list has the form \(aq@plugin\(aq, a custom
 script /usr/local/etc/smartd_warning.d/plugin is run and the word is
-removed from the list before sending mail. The string \'plugin\' may be any
-valid name except \'ALL\'.
-If \'@ALL\' is specified, all scripts in /usr/local/etc/smartd_warning.d/*
+removed from the list before sending mail.  The string \(aqplugin\(aq may be any
+valid name except \(aqALL\(aq.
+If \(aq@ALL\(aq is specified, all scripts in /usr/local/etc/smartd_warning.d/*
 are run instead.
 This is handled by the script /usr/local/etc/smartd_warning.sh
-(see also \'\-M exec\' below).
+(see also \(aq\-M exec\(aq below).
 .\" %ENDIF NOT OS Windows
 .\" %ENDIF ENABLE_SMARTDPLUGINDIR
 .\" %IF OS Windows
@@ -876,21 +876,21 @@ This is handled by the script /usr/local/etc/smartd_warning.sh
 [Windows only] If one of the following words are used as the first address
 in the comma separated list, warning messages are sent via WTSSendMessage().
 This displays message boxes on the desktops of the selected sessions.
-Address \'\fBconsole\fP\' specifies the console session only,
-\'\fBactive\fP\' specifies the console session and all active remote
-sessions, and \'\fBconnected\fP\' specifies the console session and
+Address \(aq\fBconsole\fP\(aq specifies the console session only,
+\(aq\fBactive\fP\(aq specifies the console session and all active remote
+sessions, and \(aq\fBconnected\fP\(aq specifies the console session and
 all connected (active or waiting for login) remote sessions.
 This is handled by the script EXEDIR/smartd_warning.cmd which runs
-the tool EXEDIR/wtssendmsg.exe (see also \'\-M exec\' below).
-The addresses \'\fBmsgbox\fP\' and \'\fBsysmsgbox\fP\' are now
-deprecated and have the same effect as \'\fBconsole\fP\'.
+the tool EXEDIR/wtssendmsg.exe (see also \(aq\-M exec\(aq below).
+The addresses \(aq\fBmsgbox\fP\(aq and \(aq\fBsysmsgbox\fP\(aq are now
+deprecated and have the same effect as \(aq\fBconsole\fP\(aq.
 .\" %ENDIF OS Windows
 .TP
 .B \-M TYPE
 These Directives modify the behavior of the
 \fBsmartd\fP
-email warnings enabled with the \'\-m\' email Directive described above.
-These \'\-M\' Directives only work in conjunction with the \'\-m\'
+email warnings enabled with the \(aq\-m\(aq email Directive described above.
+These \(aq\-M\(aq Directives only work in conjunction with the \(aq\-m\(aq
 Directive and can not be used without it.
 
 Multiple \-M Directives may be given.  If more than one of the
@@ -902,17 +902,17 @@ three):
 
 .I once
 \- send only one warning email for each type of disk problem detected.  This
-is the default unless state persistence (\'\-s\' option) is enabled.
+is the default unless state persistence (\(aq\-s\(aq option) is enabled.
 
 .I daily
 \- send additional warning reminder emails, once per day, for each type
 of disk problem detected.  This is the default if state persistence
-(\'\-s\' option) is enabled.
+(\(aq\-s\(aq option) is enabled.
 
 .I diminishing
 \- send additional warning reminder emails, after a one-day interval,
 then a two-day interval, then a four-day interval, and so on for each
-type of disk problem detected. Each interval is twice as long as the
+type of disk problem detected.  Each interval is twice as long as the
 previous interval.
 
 If a disk problem is no longer detected, the internal email counter is
@@ -927,7 +927,7 @@ immediately upon
 startup.  This allows one to verify that email is delivered correctly.
 Note that if this Directive is used,
 \fBsmartd\fP
-will also send the normal email warnings that were enabled with the \'\-m\' Directive,
+will also send the normal email warnings that were enabled with the \(aq\-m\(aq Directive,
 in addition to the single test email!
 
 .I exec PATH
@@ -944,7 +944,7 @@ Then it must be included in double quotes.
 By setting PATH to point to a customized script, you can make
 \fBsmartd\fP perform useful tricks when a disk problem is detected
 (beeping the console, shutting down the machine, broadcasting warnings
-to all logged-in users, etc.)  But please be careful. \fBsmartd\fP
+to all logged-in users, etc.\&)  But please be careful.  \fBsmartd\fP
 will \fBblock\fP until the executable PATH returns, so if your
 executable hangs, then \fBsmartd\fP will also hang.
 .\" %IF NOT OS Windows
@@ -953,7 +953,7 @@ Some sample scripts are included in
 .\" %ENDIF NOT OS Windows
 
 The return status of the executable is recorded by \fBsmartd\fP in
-SYSLOG. The executable is not expected to write to STDOUT or
+SYSLOG.  The executable is not expected to write to STDOUT or
 STDERR.  If it does, then this is interpreted as indicating that
 something is going wrong with your executable, and a fragment of this
 output is logged to SYSLOG to help you to understand the problem.
@@ -962,25 +962,27 @@ should send mail or write to a file or device.
 
 Before running the executable, \fBsmartd\fP sets a number of
 environment variables.  These environment variables may be used to
-control the executable\'s behavior.  The environment variables
+control the executable's behavior.  The environment variables
 exported by \fBsmartd\fP are:
 .RS 7
 .IP \fBSMARTD_MAILER\fP 4
-is set to the argument of \-M exec, if present or else to \'mail\'
+is set to the argument of \-M exec, if present or else to \(aqmail\(aq
 (examples: /usr/local/bin/mail, mail).
 .IP \fBSMARTD_DEVICE\fP 4
 is set to the device path (example: /dev/sda).
 .IP \fBSMARTD_DEVICETYPE\fP 4
-is set to the device type specified by \'-d\' directive or
-\'auto\' if none.
+is set to the device type specified by \(aq\-d\(aq directive or
+\(aqauto\(aq if none.
 .IP \fBSMARTD_DEVICESTRING\fP 4
 is set to the device description.  For SMARTD_DEVICETYPE of ata or
 scsi, this is the same as SMARTD_DEVICE.  For 3ware RAID controllers,
-the form used is \'/dev/sdc [3ware_disk_01]\'.  For HighPoint
-RocketRAID controller, the form is \'/dev/sdd [hpt_1/1/1]\' under Linux
-or \'/dev/hptrr [hpt_1/1/1]\' under FreeBSD.  For Areca controllers, the
-form is \'/dev/sg2 [areca_disk_09]\' on Linux or  \'/dev/arcmsr0 [areca_disk_09]\' on FreeBSD.  In these cases the device string
-contains a space and is NOT quoted.  So to use $SMARTD_DEVICESTRING in a
+the form used is \(aq/dev/sdc [3ware_disk_01]\(aq.  For HighPoint
+RocketRAID controller, the form is \(aq/dev/sdd [hpt_1/1/1]\(aq under Linux
+or \(aq/dev/hptrr [hpt_1/1/1]\(aq under FreeBSD.  For Areca controllers,
+the form is \(aq/dev/sg2 [areca_disk_09]\(aq on Linux or  \(aq/dev/arcmsr0
+[areca_disk_09]\(aq on FreeBSD.
+In these cases the device string contains a space and is NOT quoted.
+So to use $SMARTD_DEVICESTRING in a
 shell script you should probably enclose it in double quotes.
 .IP \fBSMARTD_DEVICEINFO\fP 4
 is set to device identify information.  It includes most of the info printed
@@ -1019,7 +1021,7 @@ one or more disk sectors could not be read.
 .br
 \fIFailedOpenDevice\fP: the open() command to the device failed.
 .IP \fBSMARTD_ADDRESS\fP 4
-is determined by the address argument ADD of the \'\-m\' Directive.
+is determined by the address argument ADD of the \(aq\-m\(aq Directive.
 If ADD is \fB<nomailer>\fP, then \fBSMARTD_ADDRESS\fP is not set.
 Otherwise, it is set to the comma-separated-list of email addresses
 given by the argument ADD, with the commas replaced by spaces
@@ -1034,17 +1036,18 @@ SMARTD_ADDRESS.
 .\" %ENDIF OS Windows
 .IP \fBSMARTD_MESSAGE\fP 4
 is set to the one sentence summary warning email message string from
-\fBsmartd\fP. 
-This message string contains space characters and is NOT quoted. So to
+\fBsmartd\fP.m46
+This message string contains space characters and is NOT quoted.  So to
 use $SMARTD_MESSAGE in a shell script you should probably enclose it in
 double quotes.
 .\" %IF NOT OS Windows
 .IP \fBSMARTD_FULLMESSAGE\fP 4
 is set to the contents of the entire email warning message string from
-\fBsmartd\fP. 
-This message string contains space and return characters and is NOT quoted. So to
-use $SMARTD_FULLMESSAGE in a shell script you should probably enclose it in
-double quotes.
+\fBsmartd\fP.
+This message string contains space and return characters and is NOT
+quoted.
+So to use $SMARTD_FULLMESSAGE in a shell script you should probably
+enclose it in double quotes.
 .\" %ENDIF NOT OS Windows
 .\" %IF OS Windows
 .IP \fBSMARTD_FULLMSGFILE\fP 4
@@ -1055,8 +1058,8 @@ the mailer or command exits.
 .\" %ENDIF OS Windows
 .IP \fBSMARTD_TFIRST\fP 4
 is a text string giving the time and date at which the first problem
-of this type was reported. This text string contains space characters
-and no newlines, and is NOT quoted. For example:
+of this type was reported.  This text string contains space characters
+and no newlines, and is NOT quoted.  For example:
 .br
 Sun Feb  9 14:58:19 2003 CST
 .IP \fBSMARTD_TFIRSTEPOCH\fP 4
@@ -1064,38 +1067,38 @@ is an integer, which is the unix epoch (number of seconds since Jan 1,
 1970) for \fBSMARTD_TFIRST\fP.
 .IP \fBSMARTD_PREVCNT\fP 4
 is an integer specifying the number of previous messages sent.
-It is set to \'0\' for the first message.
+It is set to \(aq0\(aq for the first message.
 .IP \fBSMARTD_NEXTDAYS\fP 4
 is an integer specifying the number of days until the next message will be sent.
-It it set to empty on \'\-M once\' and set to \'1\' on \'\-M daily\'.
+It it set to empty on \(aq\-M once\(aq and set to \(aq1\(aq on \(aq\-M daily\(aq.
 .RE
 .\"  The following two lines are a workaround for a man2html bug.  Please leave them.
 .\" They define a non-existent option; useful because man2html can't correctly reset the margins.
 .TP
 .B \&
-If the \'\-m ADD\' Directive is given with a normal address argument,
+If the \(aq\-m ADD\(aq Directive is given with a normal address argument,
 then the executable pointed to by PATH will be run in a shell with
 STDIN receiving the body of the email message, and with the same
 command-line arguments:
 .nf
 -s "$SMARTD_SUBJECT" $SMARTD_ADDRESS
 .fi
-that would normally be provided to \'mail\'.  Examples include:
+that would normally be provided to \(aqmail\(aq.  Examples include:
 .nf
-.B -m user@home -M exec /usr/bin/mail
-.B -m admin@work -M exec /usr/local/bin/mailto
-.B -m root -M exec /Example_1/shell/script/below
+.B \-m user@home \-M exec /usr/bin/mail
+.B \-m admin@work \-M exec /usr/local/bin/mailto
+.B \-m root \-M exec /Example_1/shell/script/below
 .fi
 
 .\" %IF OS Windows
-[Windows only] On Windows, the syntax of the \'\fBBlat\fP\' mailer is
+[Windows only] On Windows, the syntax of the \(aq\fBBlat\fP\(aq mailer is
 used:
 .nf
-- -q -subject "%SMARTD_SUBJECT%" -to %SMARTD_ADDRCSV%
+\- \-q \-subject "%SMARTD_SUBJECT%" \-to %SMARTD_ADDRCSV%
 .fi
 
 .\" %ENDIF OS Windows
-If the \'\-m ADD\' Directive is given with the special address argument
+If the \(aq\-m ADD\(aq Directive is given with the special address argument
 .B <nomailer>
 then the executable pointed to by PATH is run in a shell with
 .B no
@@ -1103,14 +1106,14 @@ STDIN and
 .B no
 command-line arguments, for example:
 .nf
-.B -m <nomailer> -M exec /Example_2/shell/script/below
+.B \-m <nomailer> \-M exec /Example_2/shell/script/below
 .fi
 If the executable produces any STDERR/STDOUT output, then \fBsmartd\fP
 assumes that something is going wrong, and a snippet of that output
 will be copied to SYSLOG.  The remainder of the output is then
 discarded.
 
-Some EXAMPLES of scripts that can be used with the \'\-M exec\'
+Some EXAMPLES of scripts that can be used with the \(aq\-M exec\(aq
 Directive are given below.
 .\" %IF NOT OS Windows
 Some sample scripts are also included in
@@ -1142,7 +1145,7 @@ SMARTD_SUBJECT and SMARTD_FULLMESSAGE
 are set by the script before running the executable.
 .TP
 .B \-f
-[ATA only] Check for \'failure\' of any Usage Attributes.  If these
+[ATA only] Check for \(aqfailure\(aq of any Usage Attributes.  If these
 Attributes are less than or equal to the threshold, it does NOT indicate
 imminent disk failure.  It "indicates an advisory condition where the usage
 or age of the device has exceeded its intended design life period."
@@ -1150,28 +1153,28 @@ or age of the device has exceeded its intended design life period."
 .TP
 .B \-p
 [ATA only] Report anytime that a Prefail Attribute has changed
-its value since the last check. [Please see the
+its value since the last check.  [Please see the
 .B smartctl \-A
 command-line option.]
 .TP
 .B \-u
 [ATA only] Report anytime that a Usage Attribute has changed its value
-since the last check. [Please see the
+since the last check.  [Please see the
 .B smartctl \-A
 command-line option.]
 .TP
 .B \-t
-[ATA only] Equivalent to turning on the two previous flags \'\-p\' and \'\-u\'.
+[ATA only] Equivalent to turning on the two previous flags \(aq\-p\(aq and \(aq\-u\(aq.
 Tracks changes in \fIall\fP device Attributes (both Prefailure and
-Usage). [Please see the \fBsmartctl\fP \-A command-line option.]
+Usage).  [Please see the \fBsmartctl\fP \-A command-line option.]
 .TP
 .B \-i ID
 [ATA only] Ignore device Attribute number \fBID\fP when checking for failure
 of Usage Attributes.  \fBID\fP must be a decimal integer in the range
-from 1 to 255.  This Directive modifies the behavior of the \'\-f\'
+from 1 to 255.  This Directive modifies the behavior of the \(aq\-f\(aq
 Directive and has no effect without it.
 
-This is useful, for example, if you have a very old disk and don\'t
+This is useful, for example, if you have a very old disk and don't
 want to keep getting messages about the hours-on-lifetime Attribute
 (usually Attribute 9) failing.  This Directive may appear multiple
 times for a single device, if you want to ignore multiple Attributes.
@@ -1179,12 +1182,12 @@ times for a single device, if you want to ignore multiple Attributes.
 .B \-I ID
 [ATA only] Ignore device Attribute \fBID\fP when tracking changes in the
 Attribute values.  \fBID\fP must be a decimal integer in the range
-from 1 to 255.  This Directive modifies the behavior of the \'\-p\',
-\'\-u\', and \'\-t\' tracking Directives and has no effect without one
+from 1 to 255.  This Directive modifies the behavior of the \(aq\-p\(aq,
+\(aq\-u\(aq, and \(aq\-t\(aq tracking Directives and has no effect without one
 of them.
 
 This is useful, for example, if one of the device Attributes is the disk
-temperature (usually Attribute 194 or 231). It\'s annoying to get reports
+temperature (usually Attribute 194 or 231).  It's annoying to get reports
 each time the temperature changes.  This Directive may appear multiple
 times for a single device, if you want to ignore multiple Attributes.
 .TP
@@ -1192,27 +1195,27 @@ times for a single device, if you want to ignore multiple Attributes.
 [ATA only] When tracking, report the \fIRaw\fP value of Attribute \fBID\fP
 along with its (normally reported) \fINormalized\fP value.  \fBID\fP must
 be a decimal integer in the range from 1 to 255.  This Directive modifies
-the behavior of the \'\-p\', \'\-u\', and \'\-t\' tracking Directives
+the behavior of the \(aq\-p\(aq, \(aq\-u\(aq, and \(aq\-t\(aq tracking Directives
 and has no effect without one of them.  This Directive may be given
 multiple times.
 
 A common use of this Directive is to track the device Temperature
 (often ID=194 or 231).
 
-If the optional flag \'!\' is appended, a change of the Normalized
+If the optional flag \(aq!\(aq is appended, a change of the Normalized
 value is considered critical.  The report will be logged as LOG_CRIT
-and a warning email will be sent if \'\-m\' is specified.
+and a warning email will be sent if \(aq\-m\(aq is specified.
 .TP
 .B \-R ID[!]
 [ATA only] When tracking, report whenever the \fIRaw\fP value of Attribute
 \fBID\fP changes.  (Normally \fBsmartd\fP only tracks/reports changes
 of the \fINormalized\fP Attribute values.)  \fBID\fP must be a decimal
 integer in the range from 1 to 255.  This Directive modifies the
-behavior of the \'\-p\', \'\-u\', and \'\-t\' tracking Directives and
+behavior of the \(aq\-p\(aq, \(aq\-u\(aq, and \(aq\-t\(aq tracking Directives and
 has no effect without one of them.  This Directive may be given
 multiple times.
 
-If this Directive is given, it automatically implies the \'\-r\'
+If this Directive is given, it automatically implies the \(aq\-r\(aq
 Directive for the same Attribute, so that the Raw value of the
 Attribute is reported.
 
@@ -1221,10 +1224,10 @@ A common use of this Directive is to track the device Temperature
 different types of system behavior affects the values of certain
 Attributes.
 
-If the optional flag \'!\' is appended, a change of the Raw
+If the optional flag \(aq!\(aq is appended, a change of the Raw
 value is considered critical.  The report will be logged as
-LOG_CRIT and a warning email will be sent if \'\-m\' is specified.
-An example is \'-R 5!\' to warn when new sectors are reallocated.
+LOG_CRIT and a warning email will be sent if \(aq\-m\(aq is specified.
+An example is \(aq\-R 5!\(aq to warn when new sectors are reallocated.
 .TP
 .B \-C ID[+]
 [ATA only] Report if the current number of pending sectors is
@@ -1234,20 +1237,20 @@ value is the Current Pending Sector count.  The allowed range of
 ID\ =\ 0.  If the \fB\-C ID\fP option is not given, then it defaults to
 \fB\-C 197\fP (since Attribute 197 is generally used to monitor
 pending sectors).  If the name of this Attribute is changed by a
-\'\-v 197,FORMAT,NAME\' directive, the default is changed to
+\(aq\-v 197,FORMAT,NAME\(aq directive, the default is changed to
 \fB\-C 0\fP.
 
-If \'+\' is specified, a report is only printed if the number of sectors
-has increased between two check cycles. Some disks do not reset this
+If \(aq+\(aq is specified, a report is only printed if the number of sectors
+has increased between two check cycles.  Some disks do not reset this
 attribute when a bad sector is reallocated.
-See also \'\-v 197,increasing\' below.
+See also \(aq\-v 197,increasing\(aq below.
 
 The warning email counter is reset if the number of pending sectors
 dropped to 0.  This typically happens when all pending sectors have
 been reallocated or could be read again.
 
 A pending sector is a disk sector (containing 512 bytes of your data)
-which the device would like to mark as ``bad" and reallocate.
+which the device would like to mark as "bad" and reallocate.
 Typically this is because your computer tried to read that sector, and
 the read failed because the data on it has been corrupted and has
 inconsistent Error Checking and Correction (ECC) codes.  This is
@@ -1266,13 +1269,13 @@ value is the Offline Uncorrectable Sector count.  The allowed range of
 ID\ =\ 0.  If the \fB\-U ID\fP option is not given, then it defaults to
 \fB\-U 198\fP (since Attribute 198 is generally used to monitor
 offline uncorrectable sectors).  If the name of this Attribute is changed
-by a \'\-v 198,FORMAT,NAME\' (except \'\-v 198,FORMAT,Offline_Scan_UNC_SectCt\'),
+by a \(aq\-v 198,FORMAT,NAME\(aq (except \(aq\-v 198,FORMAT,Offline_Scan_UNC_SectCt\(aq),
 directive, the default is changed to \fB\-U 0\fP.
 
-If \'+\' is specified, a report is only printed if the number of sectors
-has increased since the last check cycle. Some disks do not reset this
+If \(aq+\(aq is specified, a report is only printed if the number of sectors
+has increased since the last check cycle.  Some disks do not reset this
 attribute when a bad sector is reallocated.
-See also \'\-v 198,increasing\' below.
+See also \(aq\-v 198,increasing\(aq below.
 
 The warning email counter is reset if the number of offline uncorrectable
 sectors dropped to 0.  This typically happens when all offline uncorrectable
@@ -1281,7 +1284,7 @@ sectors have been reallocated or could be read again.
 An offline uncorrectable sector is a disk sector which was not
 readable during an off-line scan or a self-test.  This is important
 to know, because if you have data stored in this disk sector, and you
-need to read it, the read will fail.  Please see the previous \'\-C\'
+need to read it, the read will fail.  Please see the previous \(aq\-C\(aq
 option for more details.
 .TP
 .B \-W DIFF[,INFO[,CRIT]]
@@ -1290,21 +1293,21 @@ degrees since last report, or if new min or max temperature is detected.
 Report or Warn if the temperature is greater or equal than one of
 \fBINFO\fP or \fBCRIT\fP degrees Celsius.
 If the limit \fBCRIT\fP is reached, a message with loglevel
-\fB\'LOG_CRIT\'\fP will be logged to syslog and a warning email
-will be send if \'\-m\' is specified. If only the limit \fBINFO\fP is
-reached, a message with loglevel \fB\'LOG_INFO\'\fP will be logged.
+\fB\(aqLOG_CRIT\(aq\fP will be logged to syslog and a warning email
+will be send if \(aq\-m\(aq is specified.  If only the limit \fBINFO\fP is
+reached, a message with loglevel \fB\(aqLOG_INFO\(aq\fP will be logged.
 
 The warning email counter is reset if the temperature dropped below
 \fBINFO\fP or \fBCRIT\fP-5 if \fBINFO\fP is not specified.
 
 If this directive is used in conjunction with state persistence
-(\'\-s\' option), the min and max temperature values are preserved
-across boot cycles. The minimum temperature value is not updated
+(\(aq\-s\(aq option), the min and max temperature values are preserved
+across boot cycles.  The minimum temperature value is not updated
 during the first 30 minutes after startup.
 
 To disable any of the 3 reports, set the corresponding limit to 0.
-Trailing zero arguments may be omitted. By default, all temperature
-reports are disabled (\'-W 0\').
+Trailing zero arguments may be omitted.  By default, all temperature
+reports are disabled (\(aq\-W 0\(aq).
 
 To track temperature changes of at least 2 degrees, use:
 .nf
@@ -1324,8 +1327,8 @@ To combine all of the above reports, use:
 .fi
 
 For ATA devices, smartd interprets Attribute 194 or 190 as Temperature Celsius
-by default. This can be changed to Attribute 9 or 220 by the drive
-database or by the \'\-v 9,temp\' or \'\-v 220,temp\' directive.
+by default.  This can be changed to Attribute 9 or 220 by the drive
+database or by the \(aq\-v 9,temp\(aq or \(aq\-v 220,temp\(aq directive.
 .TP
 .B \-F TYPE
 [ATA only] Modifies the behavior of \fBsmartd\fP to compensate for some
@@ -1334,7 +1337,7 @@ multiple times.  The valid arguments are:
 
 .I none
 \- Assume that the device firmware obeys the ATA specifications.  This
-is the default, unless the device has presets for \'\-F\' in the
+is the default, unless the device has presets for \(aq\-F\(aq in the
 drive database.  Using this directive will override any preset values.
 
 .I nologdir
@@ -1360,8 +1363,8 @@ byte-reversed order.
 .I samsung3
 \- Some Samsung disks (at least SP2514N with Firmware VF100-37) report
 a self-test still in progress with 0% remaining when the test was already
-completed. If this directive is specified, \fBsmartd\fP will not skip the
-next scheduled self-test (see Directive \'\-s\' above) in this case.
+completed.  If this directive is specified, \fBsmartd\fP will not skip the
+next scheduled self-test (see Directive \(aq\-s\(aq above) in this case.
 
 .I xerrorlba
 \- This only affects \fBsmartctl\fP.
@@ -1372,19 +1375,19 @@ next scheduled self-test (see Directive \'\-s\' above) in this case.
 [ATA only] Sets a vendor-specific raw value print FORMAT, an optional
 BYTEORDER and an optional NAME for Attribute ID.
 This directive may be used multiple times.
-Please see \fBsmartctl -v\fP command-line option for further details.
+Please see \fBsmartctl \-v\fP command-line option for further details.
 
 The following arguments affect smartd warning output:
 
 .I 197,increasing
 \- Raw Attribute number 197 (Current Pending Sector Count) is not
-reset if uncorrectable sectors are reallocated.  This sets \'-C 197+\'
-if no other \'-C\' directive is specified.
+reset if uncorrectable sectors are reallocated.  This sets \(aq\-C 197+\(aq
+if no other \(aq\-C\(aq directive is specified.
 
 .I 198,increasing
 \- Raw Attribute number 198 (Offline Uncorrectable Sector Count) is not
-reset if uncorrectable sectors are reallocated.  This sets \'-U 198+\'
-if no other \'-U\' directive is specified.
+reset if uncorrectable sectors are reallocated.  This sets \(aq\-U 198+\(aq
+if no other \(aq\-U\(aq directive is specified.
 .TP
 .B \-P TYPE
 [ATA only] Specifies whether \fBsmartd\fP should use any preset options
@@ -1408,22 +1411,22 @@ The valid arguments to this Directive are:
 command-line option.]
 .TP
 .B \-a
-Equivalent to turning on all of the following Directives: 
-.B \'\-H\' 
+Equivalent to turning on all of the following Directives:
+.B \(aq\-H\(aq
 to check the SMART health status,
-.B \'\-f\' 
+.B \(aq\-f\(aq
 to report failures of Usage (rather than Prefail) Attributes,
-.B \'\-t\' 
+.B \(aq\-t\(aq
 to track changes in both Prefailure and Usage Attributes,
-.B \'\-l\ error\'
+.B \(aq\-l\ error\(aq
 to report increases in the number of ATA errors,
-.B \'\-l\ selftest\'
+.B \(aq\-l\ selftest\(aq
 to report increases in the number of Self-Test Log errors,
-.B \'\-l\ selfteststs\'
+.B \(aq\-l\ selfteststs\(aq
 to report changes of Self-Test execution status,
-.B \'\-C 197\'
+.B \(aq\-C 197\(aq
 to report nonzero values of the current pending sector count, and
-.B \'\-U 198\'
+.B \(aq\-U 198\(aq
 to report nonzero values of the offline pending sector count.
 
 Note that \-a is the default for ATA devices.  If none of these other
@@ -1448,7 +1451,7 @@ configuration file Directives might be:
 .B \-H \-l\ selftest \-l\ error \-f.
 .fi
 If you want more frequent information, use:
-.B -a.
+.B \-a.
 
 .TP
 .B ADDITIONAL DETAILS ABOUT DEVICESCAN
@@ -1465,16 +1468,16 @@ SMART properties of any devices that are found.
 which will be applied to all devices that are found in the scan.  For
 example
 .nf
-.B DEVICESCAN -m root@example.com
+.B DEVICESCAN \-m root@example.com
 .fi
 will scan for all devices, and then monitor them.  It will send one
 email warning per device for any problems that are found.
 .nf
-.B  DEVICESCAN -d ata -m root@example.com
+.B  DEVICESCAN \-d ata \-m root@example.com
 .fi
-will do the same, but restricts the scan to ATA devices only.  
+will do the same, but restricts the scan to ATA devices only.
 .nf
-.B  DEVICESCAN -H -d ata -m root@example.com
+.B  DEVICESCAN \-H \-d ata \-m root@example.com
 .fi
 will do the same, but only monitors the SMART health status of the
 devices, (rather than the default \-a, which monitors all SMART
@@ -1483,26 +1486,26 @@ properties).
 Configuration entries for specific devices may precede the \fBDEVICESCAN\fP entry.
 For example
 .nf
-.B  DEFAULT -m root@example.com
-.B  /dev/sda -s S/../.././02
-.B  /dev/sdc -d ignore
-.B  DEVICESCAN -s L/../.././02
+.B  DEFAULT \-m root@example.com
+.B  /dev/sda \-s S/../.././02
+.B  /dev/sdc \-d ignore
+.B  DEVICESCAN \-s L/../.././02
 .fi
 will scan for all devices except /dev/sda and /dev/sdc, monitor them, and run a long
-test between 2-3am every morning.  Device /dev/sda will also be monitored, but
+test between 2\(en3 am every morning.  Device /dev/sda will also be monitored, but
 only a short test will be run.  Device /dev/sdc will be ignored.
 Warning emails will be sent for all monitored devices.
 
 .TP
-.B EXAMPLES OF SHELL SCRIPTS FOR \'\-M exec\'
-These are two examples of shell scripts that can be used with the \'\-M
-exec PATH\' Directive described previously.  The paths to these scripts
-and similar executables is the PATH argument to the \'\-M exec PATH\'
+.B EXAMPLES OF SHELL SCRIPTS FOR \(aq\-M exec\(aq
+These are two examples of shell scripts that can be used with the \(aq\-M
+exec PATH\(aq Directive described previously.  The paths to these scripts
+and similar executables is the PATH argument to the \(aq\-M exec PATH\(aq
 Directive.
 
-Example 1: This script is for use with \'\-m ADDRESS -M exec PATH\'.  It appends
+Example 1: This script is for use with \(aq\-m ADDRESS \-M exec PATH\(aq.  It appends
 the output of
-.B smartctl -a
+.B smartctl \-a
 to the output of the smartd email warning message and sends it to ADDRESS.
 
 .nf
@@ -1512,16 +1515,16 @@ to the output of the smartd email warning message and sends it to ADDRESS.
 # Save the email message (STDIN) to a file:
 cat > /root/msg
 
-# Append the output of smartctl -a to the message:
-/usr/local/sbin/smartctl -a -d $SMART_DEVICETYPE $SMARTD_DEVICE >> /root/msg
- 
+# Append the output of smartctl \-a to the message:
+/usr/local/sbin/smartctl \-a \-d $SMART_DEVICETYPE $SMARTD_DEVICE >> /root/msg
+
 # Now email the message to the user at address ADD:
-/usr/bin/mail -s "$SMARTD_SUBJECT" $SMARTD_ADDRESS < /root/msg
+/usr/bin/mail \-s "$SMARTD_SUBJECT" $SMARTD_ADDRESS < /root/msg
 \fP
 .fi
 
-Example 2: This script is for use with \'\-m <nomailer> \-M exec
-PATH\'. It warns all users about a disk problem, waits 30 seconds, and
+Example 2: This script is for use with \(aq\-m <nomailer> \-M exec
+PATH\(aq.  It warns all users about a disk problem, waits 30 seconds, and
 then powers down the machine.
 
 .nf
@@ -1539,7 +1542,7 @@ EOF
 sleep 30
 
 # Power down the machine
-/sbin/shutdown -hf now
+/sbin/shutdown \-hf now
 \fP
 .fi
 
